include/lattices/cross_causal_lattice.hpp

```diff
@@ -0,0 +1,80 @@
+//  Copyright 2018 U.C. Berkeley RISE Lab
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+#ifndef SRC_INCLUDE_KVS_CROSS_CAUSAL_LATTICE_HPP_
+#define SRC_INCLUDE_KVS_CROSS_CAUSAL_LATTICE_HPP_
+
+#include "../lattices/core_lattices.hpp"
```

Comment:
?

Reply:
what's wrong with this?

Reply:
This file is in `include/lattices`. Why would we import from `../lattices` when it's the same directory we are already in?